### PR TITLE
[DO NOT MERGE] Exclude directories when extracting module wordings

### LINKED ISSUE #24987: Translations file scanning should not go to vendor folder
#### Describe the bug

If a module has a vendor folder with bit more stuff the translation view is really slow and might either timeout or run out of memory.

The translation scanning in a module should not look into `vendor` folder (and probably `lib` and some others too).

#### Expected behavior

Vendor folder should not be scanned

#### Steps to Reproduce

Create minimal module with 

```
    public function isUsingNewTranslationSystem()
    {
        return true;
    }
```

Go to bo module translation for that module
=> the translation loads nicely

Now add a composer.json with some depencies (tested with `iio/libmergepdf`) and run

```
composer install --no-dev
```

go back to the module translation page
=> the page will either be really slow, timeout or run out of memory

#### Additional information

Some slack discussion about this:
image

* PrestaShop version: develop branch
* PHP version: (whatever comes from docker-compose up in develop branch right now :)

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Coming ...
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | Fixes #24987
| How to test?      | Coming ...
| Possible impacts? | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24990)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -101,7 +101,7 @@
     "prestashop/statssales": "^2",
     "prestashop/statssearch": "^2",
     "prestashop/statsstock": "^2",
-    "prestashop/translationtools-bundle": "^4.0",
+    "prestashop/translationtools-bundle": "dev-exclude-directories",
```
Comment (sowbiba) [inline composer.json]: for testing. need to be reverted after TranslationToolsBundle release

**Comment by Progi1984** [comment]
@sowbiba Thank you for your contribution. As it's not a regression, could you target the `develop` branch ?

**Comment by PierreRambaud** [comment]
Hi @sowbiba 
Are you still working on it?

**Comment by sowbiba** [comment]
> Hi @sowbiba Are you still working on it?

Hi @PierreRambaud 
Blocked by the validation of https://github.com/PrestaShop/TranslationToolsBundle/pull/87

**Pull request closed without merging**